lua/nvim-treesitter/compat.lua

```diff
@@ -1,12 +1,17 @@
 -- Shim module to address deprecations across nvim versions
-local tsq = vim.treesitter.query
+local ts = vim.treesitter
+local tsq = ts.query
 
 local M = {}
 
 function M.get_query_files(lang, query_group, is_included)
   return (tsq.get_files or tsq.get_query_files)(lang, query_group, is_included)
 end
 
+function M.set_query(lang, query_name, text)
```

Comment:
Where is this used?

Reply:
At some point in 3rd-party plugins?

Reply:
Ideally plugins shouldn't be depending on nvim-treesitter. We are trying to reduce the scope here and put more features into core.

Reply:
Hmm, the reason for this PR as I mentioned was to PR into TSPlayground, so I will use `get_range`. I just figured that if I were to make shims for a breaking commit, I should adapt to the entire commit in its entirety.

Reply:
That is true, but if current plugins that already depend on nvim-treesitter (like playground) need it, it belongs in this module.

But I agree it makes no sense to add functions just because _some_ module _might_ need it. If and when that happens, we will add it, but not before.

Reply:
I'm sorry for wasting your time then. I'll only leave `get_range`.